PSReadLine/Completion.cs

```diff
@@ -483,17 +483,25 @@ void EnsureMenuAndInputIsVisible(IConsole console, int tooltipLineCount)
                 }
             }
 
-            public void DrawMenu(Menu previousMenu)
+            public void DrawMenu(Menu previousMenu, bool menuSelect)
             {
                 IConsole console = Singleton._console;
 
-                // Move cursor to the start of the first line after our input.
-                this.Top = Singleton.ConvertOffsetToPoint(Singleton._buffer.Length).Y + 1;
-                EnsureMenuAndInputIsVisible(console, tooltipLineCount: 0);
+                if (menuSelect)
+                {
+                    // Move cursor to the start of the first line after our input.
+                    this.Top = Singleton.ConvertOffsetToPoint(Singleton._buffer.Length).Y + 1;
+                    EnsureMenuAndInputIsVisible(console, tooltipLineCount: 0);
 
-                console.CursorVisible = false;
-                console.SaveCursor();
-                console.SetCursorPosition(0, this.Top);
+                    console.CursorVisible = false;
+                    console.SaveCursor();
+                    console.SetCursorPosition(0, this.Top);
+                }
+                else
+                {
+                    // Start a new line to show the menu contents
+                    console.Write("\n");
```

Comment:
This could write over some of the buffer if you press <kbd>Tab</kbd> anywhere that is not the last line, e.g.:
```
PS> & { Get-C<Tab><Tab>
>> }
```

After pressing <kbd>Tab</kbd> <kbd>Tab</kbd>, the screen might look like:

```
PS> & { Get-C<Tab><Tab>
Get-C1   Get-C2    Get-C3
PS> & { Get-C
>> }
```

When it should look like:

```
PS> & { Get-C<Tab><Tab>
>> }
Get-C1   Get-C2    Get-C3
PS> & { Get-C
>> }
```


```

Reply:
Thanks Jason. I submitted the fix.